Lint-Whoppers/nutshell
# Mj edit events

# Description

Got edit and save buttons to work

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1) ```git fetch --all
2) ```git checkout mj-editEvent
3)  What to check for edit and save button functionality and that object it updated in database


# Checklist:
